tfx/components/evaluator/executor.py

```diff
@@ -119,6 +119,7 @@ def Do(self, input_dict: Dict[str, List[types.Artifact]],
     # Make sure user packages get propagated to the remote Beam worker.
     unused_module_path, extra_pip_packages = udf_utils.decode_user_module_key(
         exec_properties.get(standard_component_specs.MODULE_PATH_KEY, None))
+    local_pip_package_path = []
```

Comment:
io_utils.ensure_local() returns string value, so you may want to additional list instead of using `local_pip_package_path`